tasks/install.yml

```diff
@@ -0,0 +1,79 @@
+---
+- name: Add Apt Key for NodeJS PPA
+  apt_key:
+    id: 1655A0AB68576280
+    keyserver: keyring.debian.org
+
+- name: Add NodeJS 14 PPA
+  apt_repository:
+    repo: "deb https://deb.nodesource.com/node_14.x {{ ansible_distribution_release }} main"
+    filename: nodesource
+
+- name: Install npm
+  apt:
+    name:
+      - nodejs
+
+- name: Add build user
+  user:
+    name: build-user
```

Comment:
Maybe also `system: true`, and some `comment`

Reply:
Ack